tests/js/router/index.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
+require("./router-nesting-tests");
+require("./router-params-tests");
+require("./router-static-tests");
+require("./router-wildcard-tests");
+
+// TODO:  This test causes spurious "assertion outside test context" in other tests if it's run at the same time.  Discuss with Antranig.
+//require("./router-static-failure-tests");
```

Comment:
None of the tests you have reviewed to date have used that, but I wish we had talked about it earlier, as that does seem much cleaner and lends itself to easier reuse.  My one question is how to structure it so that I can easy run a subset of tests (right now I can run the individual file that instantiates the test environment).